src/AmalgamatorSession.ts

```diff
@@ -451,6 +465,31 @@ export class AmalgamatorSession extends LoggingDebugSession {
         }
     }
 
+    protected async disassembleRequest(
+        response: DebugProtocol.DisassembleResponse,
+        args: DebugProtocol.DisassembleArguments
+    ): Promise<void> {
+        response.body = {
+            instructions: [],
+        };
+        if (args.memoryReference.indexOf('0x') === -1) {
+            const [, memoryReference] = this.disassembleHandles.get(
+                parseInt(args.memoryReference)
+            );
+            args.memoryReference = memoryReference.split(':')[1];
+            const disassemble = await this.childDaps[
+                parseInt(memoryReference.split(':')[0])
+            ].disassembleRequest(args);
+            response.body = disassemble.body;
+        } else {
+            const disassemble = await this.childDaps[
+                this.childDapIndex
+            ].disassembleRequest(args);
+            response.body = disassemble.body;
```

Comment:
The `else` should just be sending an error.

Reply:
I have updated it